functions/src/shared.ts

```diff
@@ -100,6 +100,12 @@ export function networkDocumentKey(uid: string, documentKey: string, network?: s
   return `${prefix}_${escapedKey}`;
 }
 
+export function getDocumentPath(userId: string, documentKey: string, network?: string) {
+  const networkDocKey = networkDocumentKey(userId, documentKey, network);
+  const documentPath = `documents/${networkDocKey}`;
+  return documentPath;
+}
```

Comment:
This was moved out of tree-manager.ts so it could be used by db.ts as well. I'm not sure this is the best place for it since it's not used by the functions themselves, but it seemed relatively appropriate since it uses `networkDocumentKey` (which is also defined in this file) and it's being used by things that call the `validateCommentableDocument` function.

Reply:
This seems good to me. 👍 

In another PR I'm moving this file around, but that is fine I'll handle the conflict.